pyproject.toml

```diff
@@ -73,7 +73,7 @@ unfixable = ["B"]
 force-single-line = true
 
 [tool.mypy]
-python_version              = 3.8
+python_version              = 3.9
```

Comment:
We had misunderstood what this did (it locks MyPy to this interpreter version) instead of using the current interpreter.


```suggestion
```

I suggest deleting as we did in the Python template:

https://github.com/fulcrumgenomics/python-template/pull/19